### ISSUE TITLE
No Response - Roomba i7

### ISSUE THREAD

**Issue by ccwillia** [karlvr/homebridge-roomba2#23]
### Expected Behavior
HomeKit shows current status

### Current Behavior
 Since upgrading to version v1.2.0 my Roomba i7 reports "No Response" in HomeKit.  If i click on the notification it typically disappears.

### Steps to Reproduce (for bugs)
Add Roomba i7 to accessories list

### Versions

- **Homebridge**: `v1.1.7`
- **Node**:       `node --version`
- **Plugin**:         `homebridge-roomba2 v1.2.0`

### Configuration

```
        {
            "name": "MainLevel",
            "model": "i7",
            "blid": "****",
            "robotpwd": "****",
            "ipaddress": "****",
            "autoRefreshEnabled": true,
            "keepAliveEnabled": true,
            "dockContactSensor": false,
            "accessory": "Roomba2"
        }
```

### Homebridge Log / Command Output

```
[12/31/2020, 11:05:32 AM] [LowerLevel] Battery status requested
[12/31/2020, 11:05:32 AM] [MainLevel] Battery status requested
[12/31/2020, 11:06:09 AM] [MainLevel] Running status requested
[12/31/2020, 11:06:09 AM] [MainLevel] Battery status requested
[12/31/2020, 11:06:48 AM] [MainLevel] Running status requested
[12/31/2020, 11:06:48 AM] [MainLevel] Battery status requested
[12/31/2020, 11:06:48 AM] [MainLevel] Battery level requested
[12/31/2020, 11:06:48 AM] [MainLevel] Charging status requested
```

**Comment by ccwillia** [comment]
I have reverted to version 1.1.0 and i no longer see the "no response" status warnings for my roomba i7

**Comment by FluffyPancakes** [comment]
My i7+ won't even show up on the Home app as an accessory and I've followed the instructions step by step.

**Comment by Chennyisafrag** [comment]
Mine seems to work perfectly here is my config:

{
            "name": "Miss Audrey",
            "model": "i7",
            "blid": "4***********************E",
            "robotpwd": ":1:1*******************K",
            "ipaddress": "192.168.1.63",
            "autoRefreshEnabled": true,
            "keepAliveEnabled": true,
            "dockContactSensor": true,
            "binContactSensor": true,
            "cacheTTL": 30,
            "accessory": "Roomba2"
        }

**Comment by khad** [comment]
I'm not sure that this was specific to the i7, but I also had this problem (coincidentally also with an i7). So far it appears to be fixed in 1.3.0-beta.6: https://github.com/karlvr/homebridge-roomba2/issues/75#issuecomment-926956051

@karlvr I think this issue can also be closed now that #75 is closed.

**Comment by karlvr** [comment]
@khad brilliant. Thank you. 

@ccwillia @FluffyPancakes if you're still interested, please try the latest beta of the plugin (click the wrench on the plugin and choose Install Previous Versions) and report back!
